### DISCUSSION TITLE
[Enhancement] A tutorial for people coming from WinForms

### DISCUSSION THREAD

**Discussion by ericoporto** [dotnet/maui#149] [General]
## Summary
A small tutorial page for people coming from WinForms background. Getting started, events, designer, creating a control. If I were to convert a WinForms app to use this, what should I pay attention to?

## API Changes
None.

## Intended Use Case
People with background in WinForms

**Comment by mytahir** [comment]
But I think good Documentation could solve this Problem, especially adding some Screenshot helps new to the Framework understand it better.
